src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -420,6 +423,44 @@ export default {
 				}
 			}
 		},
+
+		async handleInteraction(id) {
+			const parentId = id.split('-markdown-input-')[0]
+			const index = Array.from(document.querySelectorAll(`span[id^="${parentId}-markdown-input-"]`)).findIndex((el) => el.id.includes(id))
+			if (index === -1) {
+				return
+			}
+			let checkBoxIndex = 0
+			const lines = this.message.split('\n')
+			for (let i = 0; i < lines.length; i++) {
+				if (lines[i].trim().match(/^- {1,4}\[\s\]/) || lines[i].trim().match(/^- {1,4}\[x\]/)) {
```

Comment:
could be more that 4 spaces 
![image](https://github.com/nextcloud/spreed/assets/93392545/667441ca-4ac8-4ad2-a62a-47dac10976dc)

Reply:
between `-` and `[ ]` ?

Reply:
I assume you are referring to the spaces before `-`, `trim()` is there for that.